### PR TITLE
Add a bunch of default categories

### PR DESCRIPTION
**Pull request by anotherminh**
## Related Issues

- Part of https://transcend.height.app/T-14003

## Security Implications

_[none]_

## System Availability

_[none]_

### CODE REVIEW

**Review by michaelfarrell76 — DISMISSED** [review]

**Comment by michaelfarrell76** [comment]
oh dont forget to bump da version

File: src/subcategories/demographic.ts [hunk 1]
```diff
@@ -1,6 +1,25 @@
...
+  Industry: 'INDUSTRY',
+  /** An individual's sexual orientation */
+  SexualOrientation: 'SEXUAL_ORIENTATION',
+  /** Information about an individual's union membership */
+  UnionMembership: 'UNION_MEMBERSHIP',
+  /** Information about an individual's religious beliefs */
+  Religion: 'RELIGION',
+  /** Information about an individual's political beliefs or party membership */
+  PoliticalAffiliation: 'POLITICAL_AFFILIATION',
   // TODO: https://transcend.height.app/T-14003 - add more subcategories
```
Comment (iamtheluckyest) [inline src/subcategories/demographic.ts:23]: Remove this TODO? Or are there still more to be added?

File: src/subcategories/connection.ts [hunk 2]
```diff
@@ -1,7 +1,14 @@
...
+/**
+ * Permanent (e.g., device IDs, MAC address) or temporary (e.g., IP address)
+ * identifiers needed to establish a connection for the current browsing session.
+ */
 export const ConnectionSubCategory = makeEnum({
-  // TODO: https://transcend.height.app/T-14003 - add more subcategories
+  /** A Media Access Control address */
+  MAC_ADDRESS: 'MAC_ADDRESS',
+  /** An internet protocol address */
+  IP_ADDRESS: 'IP_ADDRESS',
```
Comment (iamtheluckyest) [inline src/subcategories/connection.ts:11]: Just curious - why do MAC_ADDRESS and IP_ADDRESS get all-capitalized keys when the others are camel cased? Later on it is written `MACAddress` and `IPAddress`, so one of those might be an oversight.

**Review by iamtheluckyest — DISMISSED** [review]

File: src/subcategories/connection.ts (hunk 2 above)
Comment (anotherminh) [reply src/subcategories/connection.ts:11]: Def an oversight! Thanks for catching

File: src/subcategories/id.ts [hunk 3]
```diff
@@ -1,7 +1,23 @@
 import { makeEnum } from '@transcend-io/type-utils';
 
+/** Information that can be used to identify an individual */
 export const IdSubCategory = makeEnum({
-  // TODO: https://transcend.height.app/T-14003 - add more subcategories
+  /** A username associated with an individual */
+  Username: 'USERNAME',
+  /** An individual's email address */
+  Email: 'EMAIL',
```
Comment (LuGary) [inline src/subcategories/id.ts:8]: I think there is already another entry under the `ContactSubCategory` enum with the exact same key and value. All of the subcategory enums are combined into one [here](https://github.com/transcend-io/privacy-types/blob/3ff1c251c4bfb823947a214b515449b56bca72f2/src/subcategories/index.ts#L18) so probably need to have a different key/value for this.

**Review by LuGary — DISMISSED** [review]

File: src/subcategories/id.ts (hunk 3 above)
Comment (anotherminh) [reply src/subcategories/id.ts:8]: good catch - i'll just remove this one

File: src/subcategories/id.ts [hunk 4]
```diff
@@ -1,7 +1,21 @@
 import { makeEnum } from '@transcend-io/type-utils';
 
+/** Information that can be used to identify an individual */
 export const IdSubCategory = makeEnum({
-  // TODO: https://transcend.height.app/T-14003 - add more subcategories
+  /** A username associated with an individual */
+  Username: 'USERNAME',
+  /** An individual's phone number */
+  PhoneNumber: 'PHONE_NUMBER',
```
Comment (LuGary) [inline src/subcategories/id.ts:8]: Is this different from the `PHONE` subcategory under `Contact`?

File: src/subcategories/location.ts [hunk 5]
```diff
@@ -1,7 +1,12 @@
 import { makeEnum } from '@transcend-io/type-utils';
 
 export const LocationSubCategory = makeEnum({
-  // TODO: https://transcend.height.app/T-14003 - add more subcategories
+  /** An address where an individual is located */
+  PhysicalAddress: 'PHYSICAL_ADDRESS',
```
Comment (LuGary) [inline src/subcategories/location.ts:5]: This is duplicated under `Contact` I think

**Review by LuGary — DISMISSED** [review]
👍 Some of the subcategories seem duplicative but maybe they're different?

File: src/subcategories/id.ts (hunk 4 above)
Comment (anotherminh) [reply src/subcategories/id.ts:8]: no i will remove

**Comment by anotherminh** [comment]
@LuGary sorry for keep pinging you but if i push new changes it dismisses your review 😅

**Review by LuGary — APPROVED** [review]
👍

**Pull request merged**